### PR TITLE
Add debug line to patch-patient.js

### PR DESCRIPTION
**Pull request by Malveen**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: azure/templates/sandbox-tests.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ steps:
       export SOURCE_COMMIT_ID=$(Build.SourceVersion)
       export STATUS_ENDPOINT_API_KEY="$(status-endpoint-api-key)"
 
-      poetry run pytest -v tests/sandbox/test_sandbox.py --junitxml=tests/sandbox-test-report.xml --reruns 3 --reruns-delay 1
+      poetry run pytest -v tests/sandbox/test_sandbox.py::TestPDSSandboxUpdateSyncWrapSuite --junitxml=tests/sandbox-test-report.xml
```
Comment (gjugpal) [inline azure/templates/sandbox-tests.yml:12]: Need to revert this change in order to run all the tests again @Malveen

**Review by PradeepVoorukonda — APPROVED** [review]
All looks good

File: sandbox/src/routes/patient/index.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 const get = require('./get-patient');
 const patch = require('./patch-patient');
-const poll = require('./polling');
+// const poll = require('./polling');
```
Comment (gjugpal) [inline sandbox/src/routes/patient/index.js:3]: Can remove this

File: sandbox/mocks/Minimal_Patient.json [hunk 3]
```diff
@@ -1,35 +0,0 @@
-{
```
Comment (gjugpal) [inline sandbox/mocks/Minimal_Patient.json:1]: This is used as a example response for a minimal patient -> see `src/services/patients.js`

Comment (Malveen) [reply sandbox/mocks/Minimal_Patient.json:1]: Added back on

**Review by gjugpal — APPROVED** [review]
